code/__secret_public.dme

```diff
@@ -14,3 +14,14 @@
 #undef SECRETS_ENABLED
 #endif
 #endif
+// BEGIN_INTERNALS
+// END_INTERNALS
+// BEGIN_FILE_DIR
+#define FILE_DIR .
+// END_FILE_DIR
+// BEGIN_PREFERENCES
+// END_PREFERENCES
+// BEGIN_INCLUDE
+#include "_placeholder.dm"
```

Comment:
gotta remove all this junk

Reply:
this is why you don't open up the secret.dme as a dme

Reply:
I didn't open it! I'm not sure how that got in there!